roles/cifmw_cephadm/templates/cert_ceph_rgw.yml.j2

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ cifmw_cephadm_cert_name }}
+  namespace: {{ cifmw_cephadm_ns }}
+spec:
+  duration: {{ cifmw_cephadm_cert_issuer_duration }}
+  issuerRef: {{ cifmw_cephadm_cert_issuer_ref }}
+  secretName: {{ cifmw_cephadm_cert_name }}
+  dnsNames:
+  - {{ cifmw_cephadm_vip }}
```

Comment:
```
2024-06-18 19:47:31.928 19 WARNING config_tempest.constants [-] Object storage swift/info API endpoint not 
discovered. Error message: HTTPSConnectionPool(host='192.168.122.2', port=8080): Max retries exceeded with 
url: /swift/info (Caused by SSLError(SSLCertVerificationError("hostname '192.168.122.2' doesn't match either of 
'192.168.122.2', 'compute-0', 'compute-1', 'compute-2'"))): urllib3.exceptions.MaxRetryError: 
HTTPSConnectionPool(host='192.168.122.2', port=8080): Max retries exceeded with url: /swift/info (Caused by
SSLError(SSLCertVerificationError("hostname '192.168.122.2' doesn't match either of '192.168.122.2', 'compute-0', 
'compute-1', 'compute-2'")))
```

Reply:
If I run Tempest with the following then I can get it to pass most most `tempest.api.object_storage` tests.
```ini
[identity]
disable_ssl_certificate_validation = True
```
I'd prefer to have a fix in DNS

Reply:
I agree, we might want to add an entry to the openstack dnsmasq and configure a forwarder in openshift-dns. 
I also think we're going to document that because this represents a real production scenario, so +1 to stay closer to what a production environment does.
I think we can also work on that in a follow up.